Location feature

aricrhodes opened the pull request:
Tests implemented along with functionality. All non-generated methods being tested.

jmartinez059 committed:
Create .gitignore

Commit:
location service created but empty

aricrhodes committed:
location domain

Commit:
Location Service repository, service tests written, working on implementing LocationServiceImpl

Commit:
LocationService class completed with testing and documentation

Commit:
LocationController test not working with RestAssured. Trouble shooting...

aricrhodes committed:
Some problems testing

Commit:
Completed Location Controller tests, 100% green light

Commit:
Completed Location Controller tests, 100% green light

Commit:
Removed one unnecessary file from the project -LoctaionControllerTests-

aricrhodes committed:
Author notes and cleanup

aricrhodes committed:
Merge branch 'location-feature' of https://github.com/revaturelabs/1808-cal-v2-location-service into location-feature

jurczakn commented on LocationService/src/main/java/com/revature/caliber/location/LocationServiceApplication.java:
```diff
@@ -0,0 +1,21 @@
+		
+	}
+	
+	@Override
```
remove CommandLineRunner if you are not using it

aricrhodes replied:
Removed in latest commit.

aricrhodes committed:
Fixed unused Command Line Runner

jurczakn reviewed (approved):
ok

aricrhodes committed:
For SonarCloud
Merge branch 'dev' of https://github.com/revaturelabs/1808-cal-v2-location-service into location-feature
